README.md

```diff
@@ -1,5 +1,9 @@
 All times are EST. Let the horror begin.
 
+##### 20 June 1015 1:59 AM
```

Comment:
It took three attempts before I realized that I was forgetting the `--force` flag on `push` to get the damn amended commit up.